osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -60,91 +60,77 @@ public void testDispatch_WithNullRequest_ThrowsNullPointerException() throws Exc
 	@Test
 	public void testDispatch_WithNonEmptyListOfVsIdsUpdateValidationSuccessful_ValidationSucceeds() throws Exception {
```

Comment:
The name is not accurate. You are not testing that the validation succeeds, the validator is mocked it should have its own tests. You are testing that **when** the validation succeeds the SFC is updated. This names gives the impression you are testing the validation which is being mocked. Maybe something like testDispatch_WithNonEmpty..._SFCIsUpdated